branchinfo.txt

```diff
@@ -4,6 +4,6 @@
 MAJOR_VERSION=1
 MINOR_VERSION=1
 PATCH_VERSION=0
-RELEASE_SUFFIX=beta
-CHANNEL=master
-BRANCH_NAME=master
+RELEASE_SUFFIX=preview1
+CHANNEL=preview
+BRANCH_NAME=release/1.1.0
```

Comment:
@dagood I take it that was FYI for me, right? Or, do you want me to correct something?